Sources/AuthFoundation/OAuth2/OAuth2Client.swift

```diff
@@ -32,16 +32,27 @@ extension OAuth2ClientDelegate {
 
 // swiftlint:disable type_body_length
 /// An OAuth2 client, used to interact with a given authorization server.
-public final class OAuth2Client {
+public final class OAuth2Client: Sendable {
     /// The URLSession used by this client for network requests.
     public let session: URLSessionProtocol
     
     /// The configuration that identifies this OAuth2 client.
     public let configuration: Configuration
     
     /// Additional HTTP headers to include in outgoing network requests.
-    public var additionalHttpHeaders: [String: String]?
-    
+    nonisolated(unsafe) public var additionalHttpHeaders: [String: String]? {
```

Comment:
What `nonisolated` means for the class?

Reply:
`nonisolated` means a property can be used from any isolation context (e.g. any thread, actor, etc).  The `unsafe` attribute is included because the property is mutable, and that we're using our own thread safety checks outside of Swift Concurrency's built-in features (e.g. in this case, a `os_unfair_lock`).